src/dex/uniswap-v2/config.ts

```diff
@@ -518,7 +518,8 @@ export const UniswapV2Config: DexConfigMap<DexParams> = {
   },
   QuickSwap: {
     [Network.POLYGON]: {
-      subgraphURL: 'https://api.fura.org/subgraphs/name/quickswap',
+      subgraphURL:
+        'https://api.thegraph.com/subgraphs/name/sameepsi/quickswap06',
```

Comment:
@mwamedacen Where did you this address? I was not able to find it in quickwapv2 docs https://docs.quickswap.exchange/

Reply:
from quickswap team directly